config/prow/cluster/pipeline_deployment.yaml

```diff
@@ -25,7 +25,7 @@ spec:
         env:
         # Use KUBECONFIG envvar rather than --kubeconfig flag in order to provide multiple configs to merge.
         - name: KUBECONFIG
-          value: "/etc/kubeconfig/config"
+          value: "/etc/kubeconfig/config:/etc/kubeconfig-build-test-infra-trusted/kubeconfig"
```

Comment:
Volume mounts are missing in this file.

Reply:
This file is not deployed as far as I can tell, reverted changes in this file to avoid confusion